Source/Core/DolphinWX/Config/PathConfigPane.cpp

```diff
@@ -147,7 +147,7 @@ void PathConfigPane::BindEvents()
   Bind(wxEVT_UPDATE_UI, &WxEventUtils::OnEnableIfCoreNotRunning);
 }
 
-void PathConfigPane::OnISOPathSelectionChanged(wxCommandEvent& event)
+void PathConfigPane::OnISOPathSelectionChanged(const wxCommandEvent& event)
```

Comment:
It would be preferable in this instance to change line 190 in this source file to

```cpp
wxCommandEvent dummy_event{};
OnISOPathSelectionChanged(dummy_event);
```

instead of making the parameters const, as wxWidgets expects non-const reference parameters for event handler functions.